app/views/users/show.html.erb

```diff
@@ -3,10 +3,8 @@
   <h2  class="bio"><span>Bio:</span><br/>
     <%= @user.bio %>
   </h2>
-  <ul>
-    <%= render @user.posts %>
-  </ul>
-  <div class="see_all_post">
-    <button type="button" class="btn btn-secondary"><%= link_to "See all user posts", user_posts_path(@user.id) %></button>
+  <%= render 'post', posts: @user.posts %>
+  <div class="all_post">
+    <button type="button" class="btn btn-secondary"><%= link_to "existing user post", user_posts_path(@user.id) %></button>
```

Comment:
- [ ] Nothing is displayed on your homepage. Kindly fix it. You will be good to go.
This is the screenshot:
![Screenshot from 2022-04-01 12-48-22](https://user-images.githubusercontent.com/65860678/161251226-88682d1d-434c-4bb9-8309-f01d6e031200.png)